### PR TITLE
Procedural images

### PR DESCRIPTION
**Pull request by nortonpmjr**
Create procedural polygons using Generators

### CODE REVIEW

File: Sources/Trickery/HexagonParameters.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import CoreGraphics
+
+struct HexagonPamaters {
```
Comment (nicorichard) [inline Sources/Trickery/HexagonParameters.swift]: I don't think this makes sense to be in Trickery? Seems like it's more for the Demo's purposes and users of our library do not need it.

Comment (nicorichard) [reply Sources/Trickery/HexagonParameters.swift]: `HexagonPamaters` => `HexagonParameters`

Comment (nicorichard) [reply Sources/Trickery/HexagonParameters.swift]: Looking above this might be duplicated

**Review by nicorichard — CHANGES_REQUESTED** [review]

**Review by nicorichard — APPROVED** [review]

File: Sources/Trickery/HexagonParameters.swift (hunk 1 above)
Comment (nortonpmjr) [reply Sources/Trickery/HexagonParameters.swift]: It shouldn't I removed it as soon I saw it

**Pull request merged**